README.md

```diff
@@ -12,11 +12,11 @@ This application was created to help users control their keyboard lights in a us
 ## Installation
 
 ### Dependencies
-The GUI requires Linux Headers, Python, Tkinter and CustomTkinter to run.
```

Comment:
If linux-headers is in fact a dependency, it should be stated that it's kernel specific (users should install headers for their kernel, e.g. Zen kernel -> `sudo pacman -S linux-zen-headers`)

Reply:
oh, ok. I'll add that quickly